terraform/azure/main.tf

```diff
@@ -13,6 +13,20 @@ module "resource_group_name" {
   suffixes = ["rg"]
 }
 
+module "app_service_plan_name" {
+  source   = "gsoft-inc/naming/azurerm//modules/web/web_app_name" // app service plan name is limited to 40 characters
```

Comment:
I couldn't find a suitable naming module for app service plans

Reply:
thats very unfortunate... may be they don't have... then we can add one ourself based on the gsoft-inc naming template like they have defined in web_app_name. 

You can create a folder called "module/app_service_name"

Reply:
I tried this, but `terraform plan` will result in the following errors then:

> │ Error: Unsupported argument
> │
> │   on main.tf line 18, in module "app_service_plan_name":
> │   18:   name     = "infra"
> │
> │ An argument named "name" is not expected here.
> ╵
> ╷
> │ Error: Unsupported argument
> │
> │   on main.tf line 19, in module "app_service_plan_name":
> │   19:   prefixes = ["orso", "arpa", "dev"]
> │
> │ An argument named "prefixes" is not expected here.
> ╵
> ╷
> │ Error: Unsupported argument
> │
> │   on main.tf line 20, in module "app_service_plan_name":
> │   20:   suffixes = ["asp"]
> │
> │ An argument named "suffixes" is not expected here.

Reply:
see my comments below for answer...